compute/k8s-services/vars.tf

```diff
@@ -531,48 +536,48 @@ variable "atlantis_image_tag" {
 
 variable "atlantis_arm_tenant_id" {
   type        = string
-  description = ""
-  default     = null
+  description = "Used to set environment variable for ARM tenant ID"
+  default     = ""
```

Comment:
Having recently started using `null` as a default for many variables myself, what is the rational for changing from `null` to an empty string?